src/backend/utilities/keypairs.ts

```diff
@@ -13,7 +12,7 @@ import {
 import { initKilt } from './initKilt';
 import { configuration } from './configuration';
 import { exitOnError } from './exitOnError';
-import { logger } from './logger';
+import { reportBalance } from './reportBalance';
```

Comment:
I’ll ask you to invert the relationship of this two modules. I have put the call to reportBalance here because the identity address was available here. But now that we have a separate module for it, please import keypairs in it to get the address, and call reportBalance from server.